README.MD

```diff
@@ -18,8 +27,31 @@ Note that we have renamed the `sandpit` environment to `dev` to align with secur
 
 Push your code to GitHub, and run the `Deploy dev` workflow. This should deploy your code to the dev environment.
 
+## Deploy to other environments
+Deployment to build, integration, staging and production is through secure pipelines. On merge to main, the stub will
+deploy to build, then if successful promote to staging, then if successful promote to integration and production.
+
 ## Adding secrets
 
 IPV have a tool that manages secrets in the stubs AWS account. If you need to add any further Secrets Manager secrets, 
 you should add the name of the secret to the externally managed list [here](https://github.com/govuk-one-login/ipv-stubs-common-infra/blob/main/utils/config-mgmt/app/configs/stubs.build.params.yaml) for non prod,
-or [here](https://github.com/govuk-one-login/ipv-stubs-common-infra/blob/main/utils/config-mgmt/app/configs/stubs.production.params.yaml) for prod.
\ No newline at end of file
+or [here](https://github.com/govuk-one-login/ipv-stubs-common-infra/blob/main/utils/config-mgmt/app/configs/stubs.production.params.yaml) for prod.
+
+## Deploying a new copy of the stub
+
+At the moment, to deploy a new copy of the stub you should duplicate the resources in the template (except the lambda authorizer,
+KMS key and Fargate cluster resources). You can see any example of how this is done by looking at the Performance Testing resources.
```

Comment:
Might be worth linking the Performance Testing resources here?

Reply:
Have updated to clarify that they're in the template - is that what you were looking for? 🙂

Reply:
Ah I just read it wrong, thought it was generic resources (i.e., documentation somewhere else not terraform resource 🤦‍♀️ )